### ISSUE TITLE
Maestral GUI windows not opening on Mac

### ISSUE THREAD

**Issue by kareila** [samschott/maestral#1028]
**Describe the bug**
The Maestral menu icon options for opening GUI windows (Show Recent Changes, Show Sync Issues, Preferences, etc.) do nothing. If I start Maestral from within the terminal, I receive the following error whenever I make the attempt:

```
Error in handler: rubicon.objc.api.ObjCInstance NSApplication has no attribute activate
Traceback (most recent call last):
  File "app_packages/toga/handlers.py", line 80, in _handler
  File "app/maestral_cocoa/app.py", line 322, in on_settings_clicked
  File "app/maestral_cocoa/settings.py", line 282, in show
  File "app_packages/toga/window.py", line 325, in show
  File "app/maestral_cocoa/private/implementation/cocoa/factory.py", line 710, in show
  File "app_packages/rubicon/objc/api.py", line 1110, in __getattr__
AttributeError: rubicon.objc.api.ObjCInstance NSApplication has no attribute activate
```

**To Reproduce**
1. Launch Terminal, type /Applications/Maestral.app/Contents/MacOS/Maestral
2. Try to activate a Maestral menu option that opens a GUI window

**Expected behaviour**
The requested information should display.

**System:**
 - Maestral version: 1.9.1
 - Python version: 3.11.5
 - OS: macOS 13.6.3 Ventura (22G436)
 - Desktop environment: n/a
 - PyQt version (for Linux GUI): n/a

**Additional context**
I have not received any update notifications since 1.8.0 - I downloaded and installed the latest version manually. That seemed to trigger a full file resync as well.

**Comment by samschott** [comment]
Thanks for the report! It appears that I've accidentally used an API that is only available starting with macOS 14.

**Comment by samschott** [comment]
I've just released a fix with v1.9.2. Before downloading from GitHub, could you try if "Check for updates" works for you this time?

**Comment by kareila** [comment]
Yes, it's working again. Thank you for the rapid response!
